src/utils/hibp.js

```diff
@@ -157,7 +157,9 @@ async function loadBreachesIntoApp (app) {
 }
 
 async function downloadBreachIcons (breaches) {
-  const breachDomains = breaches.map(breach => breach.Domain)
+  const breachDomains = breaches
+    .map(breach => breach.Domain)
```

Comment:
free tip: you could almost add `.toLowerCase()` here and avoid using it 3-4 places below on L171-186.

Reply:
Yeah I suppose so. I think I leaned this way so as not to manipulate input data (i.e. received from HIBP's API), but only data derived from it (i.e. the icon's filename). But of course, with an intermediate value like this, the line between input and output gets blurry :sweat_smile: